antora.yml

```diff
@@ -5,3 +5,9 @@ start_page: hello-world:overview.adoc
 nav:
 - modules/ROOT/nav.adoc
 server_version: '7.1.2'
+ext:
+  preview:
+    HEAD:
+      sources:
+        docs-sdk-common:
+          branches: release/7.1.2
```

Comment:
Just ooi, why this branch (4.2 / 7.1.2), and not the latest?

Reply:
I just picked the default branch in Github, maybe we should update that to 4.3? 🤦 

7.1.2 because https://github.com/couchbase/docs-sdk-nodejs/blob/release/4.2/modules/project-docs/partials/attributes.adoc#L4 defines `version-common` as that for this branch.

Reply:
Will update once we don't have to use a temp branch  :-/